### DISCUSSION TITLE
shameless plea for a release :)

### DISCUSSION THREAD

**Discussion by shawnjones253** [MichalLytek/type-graphql#1645] [General]
Sorry to nag! I'm curious if we could get a release if it's not too much trouble! ❤️

**Comment by MichalLytek** [comment]
There's only one fix in the unreleased. We need something more 😜

**Reply by shawnjones253** [reply]
Fair enough 😄

I thought I would try since that one fix would be a big help for a refactor we’re working on but I understand!

**Reply by MichalLytek** [reply]
Will try to implement something this week and make a new release 😉
